tests/server/integration/services/layers/test_layer_service.py

```diff
@@ -2,8 +2,10 @@
 import unittest
 
 from server import bootstrap_app
-from server.models.postgis.lookups import MapCategory
-from server.services.layers.layer_service import LayerService, LayerDetailsDTO
+from server.models.postgis.lookups import MapCategory, LayerType
+from server.services.layers.layer_service import LayerService, LayerDetailsDTO, LayerUpdateDTO
+
+TEST_LAYER_ID = 1
```

Comment:
Still a bit confused why this won't affect the live record with ID = 1?
Manual DB commit?